docs/app/pages/components/components-sections/tables/fixed-header-table-ng1/fixed-header-table-ng1.component.html

```diff
@@ -3,39 +3,40 @@
 <hr>
 
 <p>
-    A fixed header table is provided by the <code>scrollable-table</code> directive. The table header stays in place while
-    the table content is scrolled independently.
+    A fixed header table can be created by using the <code>fixed-header-table</code> directive. 
+    This allows the table header to remain fixed in place and visible while the table
+    content is scrolled.
+</p>
+
+<p>
+    To give a column a fixed width you should apply a class to both the <code>th</code> and associated <code>td</code> elements.
+    This class should set the <code>width</code> property and also set <code>flex: none</code>.
 </p>
 
 <p>The attributes of this directive are:</p>
 
 <uxd-api-properties>
-    <tr uxd-api-property name="watch" type="object" binding="variable" required="true">
-        Defines the values of the rows.
-    </tr>
-    <tr uxd-api-property name="container-height" type="number" binding="literal" required="true">
-        Defines the height of the scrollable container.
-    </tr>
-    <tr uxd-api-property name="header-height" type="number" binding="literal" required="true">
-        Defines the height of the fixed header.
+    <tr uxd-api-property name="tableHeight" required="true" type="number | string" binding="variable">
+        This defines the maximum height of the table.
+        When a number is provided the unit will default to pixels. 
+        If a string is provided then the unit will need to be specifed as part of the string.
```

Comment:
Sorry, could you specify that it's CSS units, and give an example here? Other than that, this all looks fine to me!

Reply:
No problem, will do that. I notice in IE it looks all wrong. The reason being is because of quirks mode in IE. Adding the doctype fixed the issue. I was wondering if perhaps I could add https://jira.autonomy.com/browse/EL-2548 to this sprint (the ticket to bring back doctype)? Apart from the review comments you have made I have only a ticket to do the contributing guide left to do, so am quite a bit ahead of schedule. If i bring this ticket into this sprint it will resolve the IE issue I am having.

Reply:
Yes, absolutely. I have a feeling that it will be a can of worms but sure if you have most of the rest of the sprint free, it would be great to get it done!

Reply:
Yeah, I would like to get it done, I think the longer it is left the more we are going to have to fix. Great, thanks!